platform/platform_tests/distance_tests.cpp

```diff
@@ -272,16 +282,19 @@ UNIT_TEST(Distance_FormattedDistance)
     {Distance(991,     Units::Feet),       990,  Units::Feet,       "990",  "990 ft"},
```

Comment:
What about tests for >1000 feet?

Reply:
They are in `Feet to Miles` section. The one where you asked about `Feet to Feet`